src/cev/components/_width_optimizer.py

```diff
@@ -0,0 +1,13 @@
+import anywidget
+
+__all__ = ["WidthOptimizer"]
+
+
+class WidthOptimizer(anywidget.AnyWidget):
+    _esm = """
+    export function render(view) {
+      setTimeout(() => {
+        view.el.parentNode.style.setProperty('--jp-widgets-inline-label-width', 'auto');
+      }, 0);
+    }
+    """
```

Comment:
can you write a comment explaining what this is doing?

Reply:
Oh sure, sorry I forgot. Usually the ipywidgets have this odd whitespace before the label starts:

<img width="1862" alt="Screenshot 2023-05-18 at 1 35 34 PM" src="https://github.com/OzetteTech/comparative-embedding-visualization/assets/84813279/4ce80c46-a5c6-4891-b185-f9fca1fd636b">

That whitespace is defined by `--jp-widgets-inline-label-width` on the labels. By unsetting the variable the whitespace is removed. (E.g., see screenshots in the PR description)